resource/ckb.toml

```diff
@@ -199,3 +199,20 @@ block_uncles_cache_size    = 30
 # [indexer_v2]
 # # Indexing the pending txs in the ckb tx-pool
 # index_tx_pool = false
+# block_filter = "block.header.number.to_uint() > \"10000000\".to_uint()"
+# cell_filter = "let script = output.type;script.code_hash == \"0xbbad126377d45f90a8ee120da988a2d7332c78ba8fd679aab478a19d6c133494\" && script.hash_type == \"data1\""
```

Comment:
we should remove this?

Reply:
Keeping these demonstrates the available features.